pages/knowledge/screen-readers/relevant-combinations/README.md

```diff
@@ -50,29 +59,43 @@ Finally, we do not have detailed information about differences between countries
 
 However: even if there are general limitations to the representativeness of the survey, it provides the best currently available data regarding the preferences of screen reader users, particularly in North America and Europe, including the UK.
 
-#### Results
+#### Results for mobile devices
+
+Allmost all (86.3%) of the respondents use a screen reader on a mobile device or tablet. That's a greater usage than on laptop (83.9%) or desktop (67.5%).
+
+The reponses to the question "Which of the following mobile screen readers do you commonly use?" show us that VoiceOver/iOS (71.2%) and TalkBack (33.0%) are by far the most popular mobile screen readers.
+
+Regarding the question, which browser is used with the user’s primary mobile screen reader, the survey results show that Safari (62.7%), Chrome (25.2%) are the prevailing mobile browsers.
+
+Respondents with disabilities used iOS devices at a higher rate than those without disabilities. For 69% of them iOS is the primary mobile/tablet platform, followed by Android (27.5%).
```

Comment:
I think it is not necessary to add `/tablet` here. Although WebAIM adds this info, we should simply speak about mobile throughout the guide (the same way we don't distinguish between desktop and laptop).